docs/reference/modules/cluster/shards_allocation.asciidoc

```diff
@@ -45,15 +45,15 @@ one of the active allocation ids in the cluster state.
      (<<dynamic-cluster-setting,Dynamic>>)
      How many concurrent outgoing shard recoveries are allowed to happen on a
      node. Outgoing recoveries are the recoveries where the source shard (most
-     likely the primary unless a shard is relocating) is allocated on the node.
+     likely the replica unless a shard is relocating) is allocated on the node.
      Defaults to `2`. Increasing this setting may cause shard movements to have
      a performance impact on other activity in your cluster, but may not make
      shard movements complete noticeably sooner. We do not recommend adjusting
      this setting from its default of `2`.
 
 `cluster.routing.allocation.node_concurrent_recoveries`::
      (<<dynamic-cluster-setting,Dynamic>>)
-     A shortcut to set both
+     A shortcut with lower precedence to set both
```

Comment:
I agree that we should say something about this here, but I think making such a small addition might not help users understand what we mean. It'd be clearer to add another sentence spelling out that this setting's value only takes effect if the more specific setting is unset.

Reply:
Makes sense. I adopted your suggestion. Thanks!